.github/workflows/MilestoneIssue.yml

```diff
@@ -16,7 +16,30 @@ jobs:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
             const TODO_COLUMN = 4971951;
-            console.log("Creating Issue card");
-            // Action will fail if a card already exists for this issue
-            github.projects.createCard({ column_id: TODO_COLUMN, content_id: context.payload.issue.id, content_type: "Issue" });
+            const IN_PROGRESS_COLUMN = 4971952;
+            const REVIEW_IN_PROGRESS_COLUMN = 4971953;
+            const REVIEW_APPROVED_COLUMN = 4971954;
```

Comment:
also declared `const VALIDATE_PEACH_COLUMN = 7838551;`?

you have it in `findCard()` but not here

Reply:
Good catch 🤦‍♂️